en/technical-reference/api-header.md

```diff
@@ -1,53 +1,67 @@
 
 As analogy in physical world, it can be considered as "a key to your car", so that only you can access your car (and only your car can be accessed by you).
 
 - For **Sandbox environment**, obtain Server Key in Sandbox Dashboard, menu: [Settings - Access Keys](https://dashboard.sandbox.midtrans.com/settings/config_info).
 - For **Production environment**, obtain Server Key in Dashboard, menu: [Settings - Access Keys](https://dashboard.midtrans.com/settings/config_info).
 
-!> Access Keys are unique for every merchant. Server Key are secret, please always **keep Server Key confidential**.
+!> Access Keys are unique for every merchant. Server Keys are secret, please always **keep Server Key confidential**.
 
 The mechanism to generate `Authorization` header value is:
-1. Follow the format of [**Basic Authentication**](https://swagger.io/docs/specification/authentication/basic-authentication/). e.g: `Username:Password`.
-2. Username and password is separated by `:` character.
-3. Server Key is used as `username`, there is no password, so password is blank/empty string.
-	- For example if your Server Key is `SB-Mid-server-abc123cde456`, then it should be `SB-Mid-server-abc123cde456:`.
-4. Encode this value into base64 format (e.g: `U0ItTWlkLXNlcnZlci1hYmMxMjNjZGU0NTY6`)
-5. Add the word `Basic ` as prefix (e.g: `Basic U0ItTWlkLXNlcnZlci1hYmMxMjNjZGU0NTY6`)
-6. Finally that is the final result of the header (e.g `Authorization: Basic U0ItTWlkLXNlcnZlci1hYmMxMjNjZGU0NTY6`)
 
-You can test with this tools to try out Authorization Header calculation:
+To generate `Authorization` header value, follow the steps given below.
+
+1. Follow the format of [**Basic Authentication**](https://swagger.io/docs/specification/authentication/basic-authentication/). (example: `Username:Password`)
+
+2. Username and password are separated by `:` character.
+
+3. *Server Key* is used as `username`, there is no password, so password is blank/empty string.
+
+  For example, if your *Server Key* is `SB-Mid-server-abc123cde456`, then `username:password` would be `SB-Mid-server-abc123cde456:`.
+
+4. Encode this value into base64 format.
+
+   For example, base64 of `SB-Mid-server-abc123cde456:` is `U0ItTWlkLXNlcnZlci1hYmMxMjNjZGU0NTY6`. 
+
+5. Add the word `Basic ` as prefix.
+
+   The above value would be `Basic U0ItTWlkLXNlcnZlci1hYmMxMjNjZGU0NTY6`.
+
+6. Your Authorization header is ready.
+
+   `Authorization: Basic U0ItTWlkLXNlcnZlci1hYmMxMjNjZGU0NTY6`
+
+You can use the below tool to try out Authorization Header calculation.
 
 <details>
 <summary><b>Authorization Header Generator</b></summary>
 <article>
-
 [Authorization Header Generator](https://jsfiddle.net/wx3hbcen/embedded/result,html/dark ':include :type=iframe width=100% height=600px')
```

Comment:
Please do not remove this `blank line` in between, because if you remove that, it can break the formatting. The `blank line` is intentional.